library/download_logfile.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
I would recommend finding a different way to do this - this shouldn't be in Github